src/mono/wasm/Wasm.Build.Tests/ProjectProviderBase.cs

```diff
@@ -55,7 +55,7 @@ public abstract class ProjectProviderBase(ITestOutputHelper _testOutput, string?
         // icu
         if (assertOptions.AssertIcuAssets)
         {
-            _testOutput.WriteLine("Skipping asserting icu assets");
+            _testOutput.WriteLine("Starting icu assets assert");
```

Comment:
Remove this line, and add `else { _testOutput.WriteLine("Skipping asserting icu assets"); }`, because printing `Starting .. ` is not helpful, but printing `Skipping icu .. ` could be useful in future.